news/_posts/2016-02-07-evb-ev3dev-lang-java-release-0.2.0.md

```diff
@@ -0,0 +1,170 @@
+---
+author: "@jabrena"
+title: "Announcing EV3Dev lang Java v0.2.0"
+---
+
+If you like Linux features from EV3Dev and you enjoy with Java, this library is the solution to develop Java software for robots on EV3Dev. [EV3Dev-lang-java](https://jabrena.github.io/ev3dev-lang-java/#/) is a Java library designed to interact with the hardware managed by EV3Dev using the LeJOS way.
+
+**What features are included in last release?**
+
+Current version includes this features:
+
+* Regulated Motor Support
+* Unregulated Motor Support
+* Sensor Support
+* Sounds
+* LeJOS Sensor filter Support
+* eSpeak TTS (Text to speech) Support
+
+JavaDocs: https://jabrena.github.io/ev3dev-lang-java/docs/api/
+
+**How to extend the features of this library?**
+
+If you have experience with Robotics, you could explore other possibilities as Computer Vision. In this case, OpenCV has a Java port and the way to install is very easy on EV3Dev:
+
+``` bash
+sudo apt-get install libopencv2.4-java
+```
+
+Using EV3Dev with Java, you have the opportunity to build on your brick directly so you only have to install your favorite Java build system and test your software or the sofware of your friend in the same way that you do on your laptop.
+
+``` bash
+sudo apt-get install git
+git clone https://github.com/jabrena/ev3dev-lang-java.git
+sudo apt-get install gradle
+sudo apt-get install maven
+sudo apt-get install ant
+cd ev3dev-lang-java/library
+mvn package
+```
+
+**What is the support for Lego Mindstorms boards?**
+
+In this release, the library has implemented the support for the EV3 Brick and their sensors & actuators included in the following kits:
+
+* [45544 LEGO MINDSTORMS Education EV3 Core Set](https://education.lego.com/es-es/lego-education-product-database/mindstorms-ev3/45544-lego-mindstorms-education-ev3-core-set)
+* [31313 LEGO MINDSTORMS EV3](http://www.lego.com/en-us/mindstorms/products/31313-mindstorms-ev3)
+
+But in the next releases, the library will add support for [Raspberry Pi](https://www.raspberrypi.org/) hats as [Mindsensors PiStorms](http://www.mindsensors.com/teaching-stem-with-robotics/13-pistorms-base-kit-raspberry-pi-brain-for-lego-robot) & [Dexter Industries BrickPi+](http://www.dexterindustries.com/brickpi/) in order to use the power of Raspberry Pi boards. 
```

Comment:
Hijo @dlech,  in tour opinion, what is the best way to talk about brickpi+ or pistons?  Shield? 

Cheers